packages/codegen/generate.server.utils.js

```diff
@@ -142,6 +142,8 @@ async function execGqlWithoutAccess (context, { query, variables, errorMessage =
 }
 
 function generateServerUtils (gql) {
+    if (!gql) throw new Error('you are trying to generateServerUtils without gql argument')
```

Comment:
the guard!

Reply:
Probably the same check is needed in client and test utilities

Reply:
In another PR